paffy parallel make failes

looks like there are not the right dependencies:

```
cd paffy
make -j  192 -O

mkdir -p ./lib ./bin
cd submodules/sonLib && PKG_CONFIG_PATH=/lib/pkgconfig:/cluster/home/markd/opt/centos7.0/x86_64/lib/pkgconfig:/usr/lib64/pkgconfig:/usr/share/pkgconfig:/cluster/home/markd/opt/centos7.0/x86_64/lib/pkgconfig:/usr/lib64/pkgconfig:/usr/share/pkgconfig: make
cd externalTools && make all
cc -Iinc -Iimpl -fsigned-char -fPIC -std=c99 -fsigned-char -O3 -g -Wall --pedantic -funroll-loops -DNDEBUG  -UNDEBUG -fopenmp -D__AVX2__ -DUSE_SIMDE -DSIMDE_ENABLE_NATIVE_ALIASES -I./inc -I./submodules/sonLib/C/inc -I./submodules/sonLib/externalTools/cutest -I./lib -I./include  -I/cluster/home/markd/opt/centos7.0/x86_64/include -DUSE_HTSLIB -o ./bin/paffy paffy_main.c impl/*.c  -L/cluster/home/markd/opt/centos7.0/x86_64/lib -lhts ./submodules/sonLib/lib/sonLib.a ./submodules/sonLib/lib/cuTest.a  -L./lib -Wl,-rpath,./lib -lz -lbz2 -lpthread -lm -lstdc++ -lm 
cc: error: ./submodules/sonLib/lib/sonLib.a: No such file or directory
cc: error: ./submodules/sonLib/lib/cuTest.a: No such file or directory
make: *** [Makefile:36: bin/paffy] Error 1
make: *** Waiting for unfinished jobs....
cd cutest && make all
cd quicktree_1.1 && make all CC="cc"
...

```

--- a/Makefile
+++ b/Makefile
@@ -26,7 +26,7 @@ ${sonLibDir}/sonLib.a : sonLib
 
 ${sonLibDir}/cuTest.a : sonLib
 
-${LIBDIR}/stPaf.a : ${libSources} ${libHeaders}  ${stPafDependencies}
+${LIBDIR}/stPaf.a : ${libSources} ${libHeaders}  ${stPafDependencies} | sonlib
 	${CC} ${CPPFLAGS} ${CFLAGS} ${LDFLAGS} -c ${libSources}
 	${AR} rc stPaf.a *.o
 	${RANLIB} stPaf.a

